index.js

```diff
@@ -55,7 +55,7 @@ var getKeyLink = function(key) {
 };
 
 var printSummary = function(hash, title, method) {
-  var res = '\n\n' + chalk[method](title + ':\n');
+  var res = '\n\n' + chalk[method](title + ':') + chalk.white('\n');
```

Comment:
mmm, not sure it should be white, why not just `\n` instead? with no color control chars?